### PR TITLE
Fix website

### PR DESCRIPTION
**Pull request by tpoterba**
cc @jbloom22 who had some opinions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,17 @@
 
 [![Zulip](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://hail.zulipchat.com?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-[Hail](https://hail.is) is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data. Hail is used throughout academia and industry as the analytical engine for major studies, projects, and services, including the Genome Aggregation Database ([gnomad.broadinstitute.org](http://gnomad.broadinstitute.org)) and Neale lab mega-GWAS ([nealelab.is/uk-biobank](https://nealelab.is/uk-biobank)).
+[Hail](https://hail.is) is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data.
 
-Unlike the Python and R scientific computing stacks, Hail:
+Hail is built to scale, and has first-class support for multi-dimensional structured data, like genomic data as in the genome-wide association study (GWAS) [tutorial](https://hail.is/docs/0.2/tutorials/01-genome-wide-association-study.html)
```
Comment (jbloom22) [inline README.md]: Hail is built to scale and has first-class support for multi-dimensional structured data, like the genomic data of a genome-wide association study (GWAS).

File: hail/www/index.md [hunk 2]
```diff
@@ -33,42 +22,33 @@ To get started using Hail:
...
 - read the [Hail Overview](https://hail.is/docs/0.2/overview.html) for a broad introduction to Hail
 - check out the [Python API](https://hail.is/docs/0.2/api.html) for detailed information on the programming interface
+- read over some of the [How-To materials](https://hail.is/docs/0.2/guides.html) for inspiration.
+
+Hail uses a continuous deployment approach to software development, which means features, bug fixes, and performance improvements land every day. We recommend updating the software frequently.
 
 ### User Support
 
-There are many ways to get in touch with the Hail team if you need help using Hail, or if you would like to suggest improvements or features. We also love to hear from new users about how they are using Hail.
+There are many ways to get in touch with the Hail team if you need help using Hail, or to suggest improvements or features.
```
Comment (jbloom22) [inline hail/www/index.md]: help using Hail or would like to suggest

**Comment by jbloom22** [comment]
I requested some minor language improvements. This is a great improvement.

**Comment by jbloom22** [comment]
And I'll create a separate science page so folks who are interested in how it's been used by others can dig in.

**Comment by tpoterba** [comment]
made the two changes

**Pull request merged**